client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -228,9 +232,9 @@ const Checkout = React.createClass( {
 
 			return purchasePaths.managePurchase( renewalItem.extra.purchaseDomain, renewalItem.extra.purchaseId );
 		} else if ( cartItems.hasFreeTrial( this.props.cart ) ) {
-			this.props.clearSitePlans( this.props.sites.getSelectedSite().ID );
+			this.props.clearSitePlans( selectedSite.ID );
 
-			return `/plans/${ this.props.sites.getSelectedSite().slug }/thank-you`;
+			return `/plans/${ selectedSite.slug }/thank-you`;
```

Comment:
@gwwar I've been a little stuck on this & other url templates that assume a selectedSite...
Should the approach be to check wether a selectedSite is available as a prop - changing the url accordingly? (if so, what to?).
In this case, it seems to me that to have bought a plan, a site must have been selected prior. 
I prefer to not make such assumptions usually and opt for being thorough but in this case I'm not so sure where the user would be redirected to in the case where they had somehow gotten here without a selectedSite.